openedx/core/djangoapps/content/learning_sequences/migrations/0003_create_course_context_for_course_specific_models.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.14 on 2020-07-22 18:27
```

Comment:
Great naming! A lot of folks leave the default generated migration name, but this is nice and clear.

Reply:
Thanks :+1: